src/polyfills.js

```diff
@@ -3,3 +3,227 @@ if (!Math.sign) {
     return (x > 0) - (x < 0) || +x
   }
 }
+
+// From : https://github.com/WebReflection/dom4/blob/master/src/event-target.js
+(function (global){
```

Comment:
why do we need this polyfill here?

Reply:
To support browsers which do not support passive events handling